scripts/github/setup/test-setup.sh

```
@@ -0,0 +1,380 @@
+        # Validate YAML syntax first
+        if ! yq eval '.' ".github/labels.yml" >/dev/null 2>&1; then
+            print_error "Invalid YAML syntax in labels.yml"
+            return 1
+        fi
+        
+        # Get label count using yq
+        label_count=$(yq 'length' ".github/labels.yml" 2>/dev/null || echo "0")
+        
+        # Check each label has required fields
+        for ((i=0; i<label_count; i++)); do
+            local name=$(yq ".[$i].name" ".github/labels.yml" 2>/dev/null | sed 's/^"//;s/"$//')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)